test/jdk/tools/jpackage/helpers/jdk/jpackage/test/WindowsHelper.java

```diff
@@ -134,16 +134,17 @@ private static boolean isUserLocalInstall(JPackageCommand cmd) {
 
     static class DesktopIntegrationVerifier {
 
-        DesktopIntegrationVerifier(JPackageCommand cmd) {
+        DesktopIntegrationVerifier(JPackageCommand cmd, String name) {
             cmd.verifyIsOfType(PackageType.WINDOWS);
             this.cmd = cmd;
+            this.name = name;
```

Comment:
I'd suggest to have the following initialization of "name" field:
`this.name = (name == null ? cmd.name() : name)`
This will help to avoid multiple `(name == null ? cmd.name() : name)` in the code.